### PR TITLE
AppsTab UX + fixes warptext on horizontal tabs

### PR DESCRIPTION
**Pull request by sean-brydon**
## What does this PR do?
Fixes no apps being installed + fixed the horizontal tab from warping on mobile	

CleanShot 2022-08-25 at 14 03 19

### CODE REVIEW

**Review by emrysal — CHANGES_REQUESTED** [review]
Should be all fine after the eslint errors are fixed 👍

File: apps/web/components/v2/eventtype/EventAppsTab.tsx [hunk 1]
```diff
@@ -70,8 +71,20 @@ export const EventAppsTab = ({
...
 
+  const installedApps = [hasPaymentIntegration, hasGiphyIntegration].filter(Boolean).length;
+
+  if (installedApps === 0) {
+    return (
+      <EmptyScreen
+        Icon={Icon.FiGrid}
+        headline={t("empty_installed_apps_headline")}
+        description={t("empty_installed_apps_description")}
+        buttonRaw={<Button href="/apps">{t("empty_installed_apps_button")} </Button>}
```
Comment (PeerRich) [inline apps/web/components/v2/eventtype/EventAppsTab.tsx:82]: it would be nice if we save event type settings here before routing

Comment (sean-brydon) [reply apps/web/components/v2/eventtype/EventAppsTab.tsx:82]: Will pick this up in a separate PR cause I agree :) - modal popop warning the user too

**Review by zomars — APPROVED** [review]
Nice work @sean-brydon 🙏🏽

**Pull request merged**